.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+<!--
+    Be sure to review our Contributing guidelines to help streamline the merging of your PR!
+
+    * PR title conventions - https://github.com/badges/shields/blob/master/CONTRIBUTING.md#running-service-tests-in-pull-requests
+    * Code formatting - https://github.com/badges/shields/blob/master/CONTRIBUTING.md#prettier
+    * Merge processes and reminders - https://github.com/badges/shields/blob/master/CONTRIBUTING.md#pull-requests
+-->
```

Comment:
I originally had leading newlines before the comment, but forgot prettier doesn't like those. My thinking was to include this as a comment so it'd be presented to folks when the open a PR without adding noise to the PR itself, and that having leading whitespace would avoid any nuisance of having to move the comments around.

Let me know if anyone thinks keeping the leading whitespace would be preferable, and I'll work on setting up an exception for pretteir

Reply:
I think it is fine as it is